azurelinuxagent/ga/agent_update_handler.py

```diff
@@ -236,7 +237,7 @@ def get_vmagent_update_status(self):
                 else:
                     status = VMAgentUpdateStatuses.Error
                     code = 1
-                return VMAgentUpdateStatus(expected_version=str(self._updater.version), status=status, code=code, message=self._last_attempted_update_error_msg)
+                return VMAgentUpdateStatus(expected_version=str(CURRENT_VERSION), status=status, code=code, message=self._last_attempted_update_error_msg)
```

Comment:
CRP expects current version running and status in the updatestatus report. The name expected_version confused me before, changing now to reflect as per protocol.

On the crp side, they check both version and status to verify the update operation status